EventsExpress/ClientApp/src/components/event/event-filter.js

```diff
@@ -33,7 +33,7 @@ class EventFilter extends Component {
                 categories: initialValues.categories,
                 statuses: initialValues.statuses,
                 radius: initialValues.radius,
-                selectedPos: initialValues.selectedPos != null ? initialValues.selectedPos : {lat: null, lng: null},
+                selectedPos: initialValues.selectedPos != null ? initialValues.selectedPos : { latitude: null, longitude: null }
```

Comment:
do we still need `selectedPos `outside the map?

Reply:
We use selectedPos only for filtering. It's just a convenient way to store the selected location along with other filter options.